autogen/agentchat/chat.py

```diff
@@ -0,0 +1,15 @@
+import logging
+from typing import Dict, List
+from dataclasses import dataclass
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
```

Comment:
Shall we use Pydantic `BaseModel`? It offers data validation for free and it is already a dependency of `openai` so we are not adding a new dependency.

Reply:
Optional.